tests/pipeline/test_domain.py

```diff
@@ -154,7 +175,9 @@ def do_checks(cls, colnames):
         do_checks(MyData, ['col1', 'col2', 'col3'])
         do_checks(MyDataSubclass, ['col1', 'col2', 'col3', 'col4'])
 
-    @parameter_space(domain_param=[US_EQUITIES, CA_EQUITIES])
+    @parameter_space(domain_param=[BE_EQUITIES, CA_EQUITIES, CH_EQUITIES,
```

Comment:
Any reason not to just use `DOMAINS` here?

Reply:
I didn't use `DOMAINS` because it includes `GB_EQUITIES`, and the test uses GB_EQUITIES as a domain meant to be different from `domain_param`

Reply:
Ah. Clearer here might be something like `BUILT_IN_DOMAINS - {GB_EQUITIES}` ? Or honestly we could probably just keep this to two or three examples. I think there are pretty hard diminishing returns on testing every single built-in domain for this.

Another option would be to use `EquitySessionDomain`s as test inputs here, since that would guarantee that we're not somehow depending on special behavior for the known built-in domains.